addons/mission/functions/fnc_bomber.sqf

```diff
@@ -24,14 +24,12 @@ params ["_unit", ["_detonateRadius", 10], ["_activateDistance", 100], ["_screami
 
 if (!is3DENPreview && {!isServer}) exitWith {};
 
-// Add vest, blacklist & set server as owner
+// Add vest
 _unit addVest "Umi_Bomb_Vest_Camo";
 
+// Should happen before any HC transfer.
```

Comment:
Does it though? What if I call this function half an hour into the mission?

Reply:
Then it will fail.